recipe/bld.bat

```diff
@@ -0,0 +1,11 @@
+REM Install the latest dotnet-core and SDK using dotnet-install scripts.
+mkdir %BUILD_PREFIX% 2> NUL
+if exist %BUILD_PREFIX%\dotnet ( rmdir %BUILD_PREFIX%\dotnet /s /q )
+curl -L -o %BUILD_PREFIX%\dotnet-install.ps1 https://dot.net/v1/dotnet-install.ps1
```

Comment:
The version of SDK that you use to build the project is mostly inconsequential in the big picture - what matters is what runtime framework the project is targeting, and that, you might guess already, depends on the project. In this case, `clr_loader` targets `net47`: https://github.com/pythonnet/clr-loader/blob/79d6c6792ebc09e43d514a3ce554b32a89a2eb19/netfx_loader/ClrLoader.csproj#L3

So a whole bunch of SDKs - not just the one particular version being used here - can target this specific runtime version. All of them produce roughly the same code for all intents and purposes, so which version we should choose is really a matter of personal preference. I chose the latest because some of the older versions can't be reliably installed with this script.

As an aside, issues like "my package worked with SDK vX but not with SDK vy" typically don't occur in the C# world, which is why tracking the exact SDK version being used isn't particularly important.

Still, I agree it would be good to track this sort of thing; at present though, there aren't that many packages that utilize the `dotnet` infrastructure (I think this is the _second_ one?). As a first step we must first agree to add `dotnet-sdk` to our defaults so that hacks like these become unnecessary.